src/pages/index.js

```diff
@@ -2,7 +2,7 @@ import React from "react"
 import { Link } from "gatsby"
 
 import Layout from "../components/layout"
-import Image from "../components/image"
+// import Image from "../components/image"
```

Comment:
I think if we're not going to use the component, it should just be deleted rather than commented out.